src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -284,9 +287,23 @@ def _resolve_default_value_from_config_file(self, arg, overrides):
         self._add_vscode_extension_metadata(arg, overrides)
 
         # same blunt mechanism like we handled id-parts, for create command, no name default
-        if self.name.split()[-1] == 'create' and overrides.settings.get('metavar', None) == 'NAME':
+        if not (self.name.split()[-1] == 'create' and overrides.settings.get('metavar', None) == 'NAME'):
+            super(AzCliCommand, self)._resolve_default_value_from_config_file(arg, overrides)
```

Comment:
Curious about the changes happened here. Looks like the logic has changed. meanwhile, the functionality of loading from config file has been disabled. Is it by design?

Reply:
np, previously if current command is `create` and parameter is `name`, it will automatically skip `az configure` value. We keep this logic.
The reason for code update here is to make _resolve_default_value_from_local_context is not  impact by above logic. Even if it is `name` of `create` command, it can also run the local context logic.